Amani-Labs/ams-api
# AMS-18  Postgres and Sequelize setup

#### Description
 - configure sequelize
 - create postgres db
 - connect sequelize to ams db

- connect GraphQl with sequelize

#### How to Test
- clone the repo

- run **yarn install**

- create db

- create .env file and add variables, check .env.example

- run **yarn db:migrate** to create tables

- run **yarn dev:watch** to start the server

- use **http://localhost:5000/** to test from your browser

#### Any background context you want to add
N/A
#### JIRA
[AMS-18](https://amanilabs.atlassian.net/browse/AMS-18)
